.vscode/launch.json

```diff
@@ -6,9 +6,12 @@
       "request": "launch",
       "name": "Run current test",
       "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
-      "args": ["--opts", "${workspaceFolder}/configs/mocha.opts", "${file}"],
+      "args": ["--opts", "${workspaceFolder}/packages/client/mocha.opts", "${file}"],
       "console": "integratedTerminal",
-      "internalConsoleOptions": "neverOpen"
+      "internalConsoleOptions": "neverOpen",
+      "env": {
+        "TS_NODE_PROJECT": "${workspaceFolder}/packages/client/tsconfig.json"
+      }
```

Comment:
Could you explain, what we need this for?

Reply:
I'm not really sure if this is really needed, I copied it from our own glsp integration, because I couldn't start the mocha tests over the launch config. But maybe the right path to the mocha config file is enough. I'll check that.

Reply:
If I try to run the test launch configs without this hint, I get a lot of errors in the log:
![image](https://user-images.githubusercontent.com/42733123/141103710-645cd42c-687c-499c-9dba-acad811a2906.png)

Reply:
I just now realized, that we are using mocha and not jest. Seems like setting the env variable is the way to go here. Please disregard my comment :slightly_smiling_face: